src/org/labkey/test/pages/study/ManageDatasetQCStatesPage.java

```diff
@@ -0,0 +1,96 @@
+package org.labkey.test.pages.study;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.components.html.Checkbox;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ManageDatasetQCStatesPage extends LabKeyPage<ManageDatasetQCStatesPage.ElementCache>
+{
+    public ManageDatasetQCStatesPage(WebDriver driver)
+    {
+        super(driver);
+    }
+
+    public static ManageDatasetQCStatesPage beginAt(WebDriverWrapper driver)
+    {
+        return beginAt(driver, driver.getCurrentContainerPath());
+    }
+
+    public static ManageDatasetQCStatesPage beginAt(WebDriverWrapper driver, String containerPath)
+    {
+        driver.beginAt(WebTestHelper.buildURL("controller", containerPath, "action"));
```

Comment:
Update this stub code:
```suggestion
        driver.beginAt(WebTestHelper.buildURL("study", containerPath, "manageQCStates"));
```